phpstan-baseline.neon

```diff
@@ -186,6 +186,12 @@ parameters:
 			count: 1
 			path: src/Serializer/SerializeVisitor.php
 
+		-
+			message: '#^Strict comparison using \=\=\= between ''\-0'' and ''0'' will always evaluate to false\.$#'
```

Comment:
This is an issue in phpstan itself: https://github.com/phpstan/phpstan/issues/12225